style/style.css

```diff
@@ -82,29 +87,201 @@ h2 {
   background-image: url(../assets/background.png);
   background-repeat: no-repeat;
   background-size: 375px 668px;
-  padding-left: 10px;
+  background-color: rgb(255, 255, 255);
   width: 375px;
   height: 718px;
+  margin: 0 auto;
+}
+
+.content ul,
+li {
+  list-style: none;
+  display: inline;
+  padding: 1px 13px;
+  margin: -3px;
```

Comment:
- [ ] Please make sure to follow the exact given design's color, font size, weight, padding, and margin on every element.
The list under the project's name (I.e. Tonic) still needs a few adjustments.
Have a look at the following two images and make the corresponding changes.
### **Your design**
![works](https://user-images.githubusercontent.com/47625407/120863333-a8052400-c58a-11eb-86d8-5526d2680450.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)**
![originalworks](https://user-images.githubusercontent.com/47625407/120863268-8b68ec00-c58a-11eb-833b-29885a583d3b.png)